pkg/execution/executor/validate.go

```diff
@@ -80,7 +80,14 @@ func (r *runValidator) checkStepLimit(ctx context.Context) error {
 	if limit > 0 && r.md.Metrics.StepCount >= limit {
 		// Create a new driver response to map as the function finished error.
 		resp := state.DriverResponse{}
-		resp.SetError(state.ErrFunctionOverflowed)
+
+		gracefulErr := state.StandardError{
+			Error:   state.ErrFunctionOverflowed.Error(),
+			Name:    "ErrFunctionOverflowed",
```

Comment:
Can we make this name a `const`? atm I'm starting to:
- Add these errors over here: https://github.com/inngest/inngest/blob/b32061954f142aa79c247fc26ac3cb07b225ddff/pkg/execution/execution.go#L22
- Prefix them with `Inngest` so users can easily differentiate these from their own errors (and we can more easily provide UI help)